src/bonsai/rest/validatorAccountMetadata.ts

```diff
@@ -92,3 +98,56 @@ export function setUpAccountFeeTierQuery(store: RootStore) {
     store.dispatch(setAccountFeeTierRaw(loadableIdle()));
   };
 }
+
+export function setUpRewardsParamsQuery(store: RootStore) {
+  const cleanupEffect = createValidatorQueryStoreEffect(store, {
+    name: 'rewardsParams',
+    selector: () => true,
+    getQueryKey: () => ['rewardsParams'],
+    getQueryFn: (compositeClient) => {
+      return () => compositeClient.validatorClient.get.getRewardsParams();
+    },
+    onResult: (result) => {
+      store.dispatch(
+        setRewardsParams(
+          mapLoadableData(queryResultToLoadable(result), (d) => parseToPrimitives(d).params)
+        )
+      );
+    },
+    onNoQuery: () => store.dispatch(setRewardsParams(loadableIdle())),
+    refetchInterval: timeUnits.hour,
+    staleTime: timeUnits.hour,
+  });
+  return () => {
+    cleanupEffect();
+    store.dispatch(setRewardsParams(loadableIdle()));
+  };
+}
+
+export function setUpRewardsTokenPriceQuery(store: RootStore) {
+  const cleanupEffect = createValidatorQueryStoreEffect(store, {
+    name: 'rewardsParamsTokenPrice',
+    selector: (state) => state.raw.rewards.data.data?.marketId,
+    getQueryKey: (market) => ['rewardsParamsTokenPrice', market],
+    getQueryFn: (compositeClient, market) => {
+      if (market == null) {
+        return null;
+      }
+      return () => compositeClient.validatorClient.get.getPrice(market);
+    },
+    onResult: (result) => {
+      store.dispatch(
+        setRewardsTokenPrice(
+          mapLoadableData(queryResultToLoadable(result), (d) => parseToPrimitives(d).marketPrice)
+        )
+      );
+    },
+    onNoQuery: () => store.dispatch(setRewardsTokenPrice(loadableIdle())),
+    refetchInterval: timeUnits.hour,
+    staleTime: timeUnits.hour,
+  });
+  return () => {
+    cleanupEffect();
+    store.dispatch(setRewardsTokenPrice(loadableIdle()));
+  };
+}
```

Comment:
feels like these deserve their own rest file. it doesn't have any relation with Account